sw/device/silicon_creator/lib/ownership/owner_block.c

```diff
@@ -528,6 +528,41 @@ rom_error_t owner_block_info_lockdown(const owner_flash_info_config_t *info) {
   return kErrorOk;
 }
 
+rom_error_t owner_block_info_isfb_erase_enable(
+    boot_data_t *bootdata, const owner_config_t *owner_config) {
+  if (bootdata->ownership_state != kOwnershipStateLockedOwner)
+    return kErrorOk;
+  if ((hardened_bool_t)owner_config->isfb == kHardenedBoolFalse)
+    return kErrorOk;
+  if ((hardened_bool_t)owner_config->info == kHardenedBoolFalse)
```

Comment:
Why is this check repeated from line 535 to 537?

Reply:
I thought it was a duplicate at first too :)

The first checks that `isfb` exists, the second checks that `info` (flash INFO) exists.

Reply:
haha, I re-read those lines of code so many times before I made the comment even.

What do you think about adding a `// comment` saying what each does so it will be easier to detect the 3 character difference in those lines?